src/test/resources/com/powsybl/config/test/config.yml

```diff
@@ -0,0 +1,11 @@
+dynawaltz:
+  homeDir: /home/runner/work/dynamic-simulation-server/dynawo
```

Comment:
is this used ?

Reply:
Yes, homDir is mandatory and loaded by DynaWaltzConfig for the DynaWaltzProvider.

In prod env, config.yml is in the config directory ~/.itool

Reply:
In the next step, we will test with container.. so this config can be removed?

Reply:
Is it used by Github CI ?

Reply:
yes! it is used when launching tests.